src/App.js

```diff
@@ -1,14 +1,10 @@
 import './App.css';
+import Calculator from './Components/Calculator/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <p>
-          Math Magicians.
-        </p>
-
-      </header>
+    <div>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] According to the requirement, The App.js component should be class-based. 👌

![image](https://user-images.githubusercontent.com/45231276/188875200-bfa346be-0eaf-4266-b343-381ede6d9a48.png)